### DISCUSSION TITLE
Getting API Errors

### DISCUSSION THREAD

**Discussion by BlueAnchorNM** [wazuh/wazuh#23812] [General]
Hi Team -- I'm getting API errors when I navigate sections of the Wazuh platform, for example anything related to Rules. 
The errors read like this:

**RequestError: Error fetching items
Wazuh API error: ERR_BAD_RESPONSE - Wazuh Internal Error**

Full error:
RequestError: Wazuh API error: ERR_BAD_RESPONSE - Wazuh Internal Error
    at settle (https://myIP/47502/bundles/plugin/wazuh/wazuh.plugin.js:8:20234)
    at XMLHttpRequest.onloadend (https://MYIP/47502/bundles/plugin/wazuh/wazuh.plugin.js:8:25708)

Also this error:

**AxiosError
Error updating file content: Wazuh API error: ERR_BAD_RESPONSE - Wazuh
Internal Error**

Help please.

**Comment by BlueAnchorNM** [comment]
I actually fixed this. I had to delete custom rules, there was one there about "syscheck" that I think was messing with the config.

**Reply by MarantosGeorge** [reply]
Hi, Can you please explain what was the issue you found, and what you did to fix it?
I am running into the same issue at the moment.
